### ISSUE TITLE
Unable to flash 7C81 via EPP

### ISSUE THREAD

**Issue by hachi** [LinuxCNC/mesaflash#70]
I have a 7C81 that I flashed with 7c81epp.bit and now needs to be flashed via EPP mode.

I was having trouble so I uncommented some of the debugging printfs in the epp .c file for mesaflash.

With the device unpowered but attached the following happens:

```
root@mora:~/mesaflash# ./mesaflash --device 7C81 --epp --verbose --addr 0x378 --readhmid
wrote control 0x04
read status 0x78
selected address 0x8000
read data 0x80808080
read status 0x78
selected address 0x8100
read data 0x81818181
read status 0x78
No 7C81 board found
```

And then powering it on:

```
root@mora:~/mesaflash# ./mesaflash --device 7C81 --epp --verbose --addr 0x378 --readhmid
wrote control 0x04
read status 0xF8
selected address 0x8000
read data 0xFFFFFFFF
read status 0xF8
selected address 0x8100
read data 0x55AACAFE
read status 0xF8
selected address 0x8000
read data 0xFF
read data 0xFF
read data 0xFF
read data 0xFF
read status 0xF8
selected address 0x8100
read data 0xFE
read data 0xCA
read data 0xAA
read data 0x55
read status 0xF8
selected address 0x810C
read data 0x00
read data 0x04
read data 0x00
read data 0x00
selected address 0x840C
read data 0x4D
read data 0x45
read data 0x53
read data 0x41
read data 0x37
read data 0x43
read data 0x38
read data 0x31
No 7C81 board found
```

It clearly sees the card... the hex string 4D 45 53 41 37 43 38 31 is "MESA7C81" which is the fingerprint for the card.

I'm looking at the code to see if it's something simple I can "fix" but I figured I should report this.

**Comment by hachi** [comment]
Oh the epp needs a board definition for 8C71 to work.

PR incoming

**Comment by SebKuzminsky** [comment]
Awesome!  I'll keep an eye out for the PR :-)

**Comment by SebKuzminsky** [comment]
@hachi reports this issue is fixed by #71 .
